src/components/Conversations/ConversationsMessages.tsx

```diff
@@ -0,0 +1,82 @@
+import { useRouter } from 'next/router';
+import { useState, useEffect } from 'react';
+import { serviceClient } from '~/utils/trpc';
+import { TextInput, Group, Button, Loader, Center, createStyles } from '@mantine/core';
+import { IconSend } from '@tabler/icons-react';
+import { ConversationsMessage } from './ConversationsMessage';
+
+export function ConversationMessages() {
+  const router = useRouter();
+  const { classes } = useStyles();
+  const [messages, setMessages] = useState<any>([]);
```

Comment:
Avoid using `any` as much as posisble, ideally you can export/expose the typing of the messages.